### ISSUE TITLE
Docker proxy ignores targetPort

### ISSUE THREAD

**Issue by Sakorah** [F5Networks/f5-appsvcs-extension#34]
##### ISSUE TYPE
 - Bug Report

##### AS3 BUILD/ VERSION
```
~$ curl -k https://localhost:8443/mgmt/shared/appsvcs/info
{"version":"3.5.0","release":"3","schemaCurrent":"3.5.0","schemaMinimum":"3.0.0"}
```

##### BIGIP VERSION
```
Sys::Version
Main Package
  Product     BIG-IP
  Version     13.1.1.2
  Build       0.0.4
  Edition     Point Release 2
  Date        Thu Oct 11 15:32:21 PDT 2018
```

##### SUMMARY
```
Using the AS3 in a Docker container accessing a BIG-IP in AWS fails with the management port listening on port 8443. AS3 tries to connect to port 443.
```

##### STEPS TO REPRODUCE
```
{
  "class": "AS3",
  "action": "deploy",
  "persist": true,
  "targetHost": "MGMT-IP",
  "targetPort": 8443,
  "targetUsername": "XXX",
  "targetPassphrase": "XXX",
  "declaration": {
    "class": "ADC",
    "schemaVersion": "3.5.0",
...
```

##### EXPECTED RESULTS
Should connect to port 8443 as configured in the json file.

##### ACTUAL RESULTS
```
~$ curl -sku admin:admin -H "Content-Type: application/json" -X POST https://localhost:8443/mgmt/shared/appsvcs/declare --data-binary "@as3.json";
{"code":500,"declarationFullId":"","message":"cannot contact xx.xx.xx.xx (GET https://xx.xx.xx.xx:443/mgmt/shared/echo failed (connect ECONNREFUSED xx.xx.xx.xx:443))"}
```

**Comment by lephubui** [comment]
Hi @Sakorah , thank you for the feedback.
We have noticed this bug on AS3 container 3.5.0, and the new coming version 3.6.0 will solve it (which will release soon). Stay tuned on it.

**Comment by lephubui** [comment]
Hi @Sakorah, we have published the new AS3 docker image version 3.6.0 (associated with latest tag) for the release today that solve your issue.
https://hub.docker.com/r/f5devcentral/f5-as3-container/ 
Please let us know if there are other issues or you can find us through the Slack chance **f5-appsvcs-extention**.

**Comment by kaustriaf5** [comment]
Fixed with 3.6.0, closing this issue.
